server.go

```diff
@@ -1199,7 +1231,7 @@ func padBase64(s string) string {
 	return s + strings.Repeat("=", padding)
 }
 
-//Takes in an int and returns a string with thousands separators
+//Comma formats a number as a string with thousands separators
```

Comment:
idiomatic code documentation (used by GoDoc) is to have the function, type, or variable name as the first token of the comment. As the language is strictly typed, we don't need to call out the types. For a utility function like this, it is hard to not specifically call out the types. (shrug)